README.md

```diff
@@ -89,6 +89,12 @@ cellxgene prepare --help
 pip install cellxgene[louvain]
 ```
 
+On some environments installing the extra requires for lovain as above does not work. Alternatively, you can install these dependencies by hand in addition to the cellxgene package.
```

Comment:
Suggest rewording:   

If the aforementioned optional package installation fails, you can also install these packages directly.  For example, `pip install louvain python-igraph`.

Reply:
+1

Reply:
👍